update_sources.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import argparse
+import shutil
+import subprocess
+
+git_repo_path = os.path.abspath(subprocess.check_output(['git', 'rev-parse', '--show-toplevel']).strip())
```

Comment:
This fork needs to track updates in the real MobileCoin SDK.  This script can be used to copy over the files from the real SDK.  It _does not_ update the podspec as well; that must be done manually.